shell/src/main/java/org/crsh/lang/impl/script/ScriptCompiler.java

```diff
@@ -163,7 +164,7 @@ public void open(CommandContext<? super Object> consumer) {
                   public void close() throws IOException, CommandException {
 
                     // Execute sequentially the script
-                    BufferedReader reader = new BufferedReader(new InputStreamReader(new ByteArrayInputStream(source)));
+                    BufferedReader reader = new BufferedReader(new InputStreamReader(new ByteArrayInputStream(source), Charset.forName("UTF-8")));
```

Comment:
It would be better if this was settable, rather than forcing an encoding, especially if the default charset is not desired.